README.md

```diff
@@ -87,16 +86,19 @@ This project supports below scenarios for end-to-end guest operating system vali
 | UnionTech OS Server 20 1050a                    | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
 | Fedora Server 36 and later                      | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
 | ProLinux Server 7.9, 8.5                        | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
+| FreeBSD 13 and later (*)                            | :heavy_check_mark:               |                          |                                    |
```

Comment:
"Existing VM with Guest Operating System Installed" is not checked because not all the test cases passed, right?

Reply:
FreeBSD doesn't support any test case except deploy_vm. So it couldn't run on existing VM.